src/library_syscall.js

```diff
@@ -200,112 +193,117 @@ var SyscallsLibrary = {
 #endif // NO_FILESYSTEM == 0
     get64: function() {
       var low = SYSCALLS.get(), high = SYSCALLS.get();
-      if (low >= 0) assert(high === 0);
-      else assert(high === -1);
+      SYSCALLS.assert64(low, high);
 #if SYSCALL_DEBUG
       Module.printErr('    (i64: "' + low + '")');
 #endif
       return low;
     },
     getZero: function() {
-      assert(SYSCALLS.get() === 0);
+      SYSCALLS.assertZero(SYSCALLS.get());
+    },
+    assertZero: function(zero) {
```

Comment:
Ah, yeah that is definitely too much. I was mechanically changing `foo = getX()` to `assertX(foo)`, which doesn't really make sense here.